### ISSUE TITLE
Login error: "fail" status, message "feedback_required".

### ISSUE THREAD

**Issue by mpgraber** [instaloader/instaloader#1099]
In Python, as of yesterday, I'm getting this new error using instaloader.  I can log in to instagram in a browser with no problem - no challenge to the account credentials or suspicious activity, but I am getting the prompt to save my password (by Instagram).  I generally select "No."  Do I have to select "Yes" to have instaloader not receive this error?

**Comment by Showmaster3000** [comment]
I think its similar to #929 or #989.
Please have a look there an give feedback.

**Comment by mpgraber** [comment]
@Showmaster3000 thank you for those tips.  I'm not sure it's related.  I just waited 24 hours and tried again.  No code changes and everything works fine.  I'm using 4 accounts in a multithreaded python script.  All of them had the "feedback required" prompt upon login, and instaloader was bailing, but no longer.  I did not change passwords, nor did I select "Yes" (in Chrome) to have instagram.com save my the password for any account.  Very odd behaviour.
